Sources/Public/Animation/LottieView.swift

```diff
@@ -17,22 +17,26 @@ public struct LottieView<Placeholder: View>: UIViewConfiguringSwiftUIView {
     placeholder = nil
   }
 
-  /// Creates a `LottieView` that displays the given `DotLottieFile`
+  /// Initializes a `LottieView` with the provided `DotLottieFile` for display.
+  ///
+  /// - Important: Avoid using this initializer with the `SynchronouslyBlockingCurrentThread` APIs.
+  ///              If decompression of a `.lottie` file is necessary, prefer using the `.init(_ loadAnimation:)`
+  ///              initializer, which decompresses the file asynchronously.
```

Comment:
Including an example may be helpful here: 

```suggestion
  /// - Important: Avoid using this initializer with the `SynchronouslyBlockingCurrentThread` APIs.
  ///              If decompression of a `.lottie` file is necessary, prefer using the `.init(_ loadAnimation:)`
  ///              initializer, which takes an asynchronous closure:
  ///              
  ///                LottieView {
  ///                  try await DotLottieFile.asset(named: myDotLottieFile)
  ///                }
  ///
```